commands/github/commands.go

```diff
@@ -242,14 +242,25 @@ func fetchTeamRepos() error {
 	fmt.Print("Fetch from remotes?(Y/N): ")
 	text, _ := reader.ReadString('\n')
 	if strings.Contains(text, "Y") {
-
+		_, er := os.Stat(".gitmodules")
+		isMetaRepo := !os.IsNotExist(er)
```

Comment:
Do you want to refactor the fetchRepo function also and/or create a generic function that suites both team and personal? ;)